_posts/2024-12-27-documentos-estadoSII.md

```
@@ -0,0 +1,18 @@
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [ riesgo, créditos , clientes, ejecutivos, operaciones ]
+comments: false
+author: Erick Burgos
+---
+
+## Ahora es más fácil verificar el estado del SII en los documentos de Advance
+
++Puedes consultar directamente el estado del SII en la sección de `Documentos` dentro de `Factoring`. Además, podrás filtrar los resultados por el estado del SII para efectuar búsquedas más ágiles. Al posicionar el cursor sobre cada estado, podrás obtener más detalles de manera rápida y sencilla.
+
+<video width="100%"  controls autoplay loop>
```

Markdownlint (0.37.0)

Element: video
Inline HTML

(MD033, no-inline-html)